@xen-orchestra/fs/src/fs.spec.js

```diff
@@ -0,0 +1,121 @@
+/* eslint-env jest */
+
+import { getHandler } from '.'
+import fs from 'fs-extra'
```

Comment:
Don't use `fs`!!!

Figure out a way to test consistency only by using `xo/fs` APIs.